tests/runtests.sh

```diff
@@ -207,13 +222,24 @@ ${SCRIPT_DIR}/smoketests.py --toaster_url="$toastername:8000" \
                 $timeout_arg \
                 $pokybranch_arg
 echo "smoketests PASSED!"
-${SCRIPT_DIR}/checkartifacts.sh $tempdir/toasterbuild/build-toaster-2
+
+${SCRIPT_DIR}/checkartifacts.sh $tempdir/$toasterbuild/build-toaster-2
 echo "checkartifacts PASSED!"
 
-grep -q "No migrations to apply" $toasterlog
-echo "no migration PASSED"
+grep -q "No migrations to apply" $toasterlog || test "" != $LOCALDIR
+echo "no migration priming PASSED"
 
+if grep -q "INFO Fetching layers" $toasterlog; then
+    if [ "" == $LOCALDIR ]; then
+	exit 1
```

Comment:
I think the error handling needs to also trap exit, or this should be changed to be "false" instead of "exit 1".